### ISSUE TITLE
How does the plugin uses the default weight?

### ISSUE THREAD

**Issue by uhlhosting** [Zasilkovna/WooCommerce#193]
We had our products in past without weight yet the module had the option to add a minimum value that would be sent to packeta.

Can you explain how do this works now?

Is it a must to have weight on all products?

**Comment by MatyasKoval** [comment]
Hello @uhlhosting,
Packeta as a whole started to require weight as a parameter of parcels last year, due to the requirements of our carriers and to improve our prediction capabilities in terms of the carrier vehicle capacity etc.

That is the main reason weight became required - therefore having a minimum/default value, that would be filled in if the order did not have any weight would be contradictory to the reson weight became required in the first place.
Since we would not be getting the required information, what is worse we would be getting wrong information, which we would then use for the capacity predictions.

The way it works now is that the order weight is computed from the weight of the products in the order + packaging weight you have set up in the plugin settings (the packaging weight only gets added when the weight is non-0). This weight also influences the prices of the carriers you have set up according to their weight rules.

In short, if you would like to have weight computed automaticaly, you have to have weight set up for each of your products.

Best regards,
Matyáš Kovaľ - IT support

**Comment by uhlhosting** [comment]
Can you show me where in menu of packeta goes this 

"packaging weight you have set up in the plugin settings (the packaging weight only gets added when the weight is non-0)."

Because we added now the weight to our products. Would need to know what else we would need to have this in smooth conditions.

**Comment by uhlhosting** [comment]
The values would be 0.3 for 300 grams? or 0,3 for 300 grams? should be  used , or .?

**Comment by MatyasKoval** [comment]
Hello @uhlhosting ,
as far as I know, the textbox in the settings will not even allow you to write the "," sign, so the only option is 0.3 and that is the correct option.

Best regards,
Matyáš Kovaľ - IT support
